worker/workercmd/dns.go

```diff
@@ -22,7 +22,8 @@ func (config DNSConfig) Server() (*dns.Server, error) {
 		for _, server := range resolvConf.Servers {
 			response, _, err := client.Exchange(r, fmt.Sprintf("%s:%s", server, resolvConf.Port))
 			if err == nil {
-				w.WriteMsg(response)
+				response.Compress = true
```

Comment:
Is there any problem with this unconditionally being set to `true`? I didn't know DNS compression was a thing, but I guess everything should be compatible with it; looks like it's been around for [a while](https://www.ietf.org/rfc/rfc1035.txt).

I guess we have to explicitly set this because the library transparently handled the compression and gave us a `*Msg`, but since we're going to re-transmit it we need to tell it to compress it again.

Reply:
私は2つの理由で大丈夫だと認識しています。
* Dockerの内部DNSも常に圧縮をしているため。
https://github.com/docker/libnetwork/blob/master/resolver.go#L537
* ライブラリの内部で圧縮可能かどうか検査しているため
https://github.com/miekg/dns/blob/master/msg.go#L767

Reply:
sounds good! i'll go ahead and merge